### PR TITLE
Ci for ruby 2.7 and 3.0

### PR DESCRIPTION
**Pull request by cosmo0920**
And I also added a commit which prevents overflow for long long unsigned int hex digit.

### CODE REVIEW

File: ext/winevt/winevt_utils.cpp [hunk 1]
```diff
@@ -267,7 +267,12 @@ extract_user_evt_variants(PEVT_VARIANT pRenderedValues, DWORD propCount)
         rb_ary_push(userValues, rbObj);
         break;
       case EvtVarTypeHexInt64:
-        rbObj = rb_sprintf("%#I64x", pRenderedValues[i].UInt64Val);
```
Comment (cosmo0920) [inline ext/winevt/winevt_utils.cpp:270]: Ruby's `rb_sprintf` does not support `%#I64x` format string.
We should use hand written format code here: https://github.com/fluent-plugins-nursery/winevt_c/pull/35/files#diff-7d3a7f81db65ff315b6f1e9e118b13ac952f66cc81fae2f05b10c8c6a4c69ac8R270-R275

File: ext/winevt/winevt_utils.cpp [hunk 2]
```diff
@@ -267,7 +267,12 @@ extract_user_evt_variants(PEVT_VARIANT pRenderedValues, DWORD propCount)
...
+        uint32_t  high;
+        uint32_t  low;
+
+        high = pRenderedValues[i].UInt64Val >> 32;
+        low = pRenderedValues[i].UInt64Val & 0x00000000FFFFFFFF;
+        rbObj = rb_sprintf("0x%04x%08x", high, low);
```
Comment (ashie) [inline ext/winevt/winevt_utils.cpp:275]: Why higher part is `04x` length?

Comment (cosmo0920) [reply ext/winevt/winevt_utils.cpp:275]: Hmm, we should consider using hex length....

Comment (cosmo0920) [reply ext/winevt/winevt_utils.cpp]: I also used a fixed length for high-order bit in https://github.com/fluent-plugins-nursery/winevt_c/pull/35/commits/0c4c1be0c3d6ffd01e5fcfa3998d67ffd054fa17.

**Review by ashie — APPROVED** [review]

**Pull request merged**